### PR TITLE
UOT 103594

### PR DESCRIPTION
**Pull request by cskiz**
- Adds additional entities to improve entity linking
- Move CLIs to their own directory
- Cleanup / re-factor

### CODE REVIEW

File: gamechangerml/src/text_classif/tests/concat_df.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import fnmatch
+import pandas as pd
+import gamechangerml.src.text_classif.utils.log_init as li
+
+logger = logging.getLogger(__name__)
+
+
+if __name__ == "__main__":
+    li.initialize_logger(to_file=False, log_name="none")
+    failed_data_path = "/Users/chrisskiscim/projects/failed_docs"
```
Comment (cWren0110) [inline gamechangerml/src/text_classif/tests/concat_df.py:12]: Remove local path if possible.

**Review by cWren0110 — CHANGES_REQUESTED** [review]

File: gamechangerml/src/text_classif/cli/predict_table.py [hunk 2]
```diff
@@ -188,8 +188,8 @@ def predict_table(
     )
     parser.add_argument(
         "-a",
-        "--agencies-path",
-        dest="agencies_path",
+        "--orgs-file",
```
Comment (jgrundy) [inline gamechangerml/src/text_classif/cli/predict_table.py:191]: Can 'orgs-file' be added as a separate argument? Removing agencies-path will break one of the util functions.

**Review by jgrundy — CHANGES_REQUESTED** [review]

File: gamechangerml/src/text_classif/cli/predict_table.py (hunk 2 above)
Comment (cskiz) [reply gamechangerml/src/text_classif/cli/predict_table.py:191]: Got it. I'll make the change.

Comment (cskiz) [reply gamechangerml/src/text_classif/cli/predict_table.py:199]: See the commit below

File: gamechangerml/src/text_classif/tests/concat_df.py (hunk 1 above)
Comment (cskiz) [reply gamechangerml/src/text_classif/tests/concat_df.py]: Resolved in the latest commit.

**Review by jgrundy — APPROVED** [review]

**Review by cWren0110 — APPROVED** [review]

**Pull request merged**